docs/Hosts/Developer-Workstations.md

```diff
@@ -46,16 +46,15 @@ checked out and running on workstations, though this feature is not
 necessarily dependent on workstations, and could be used on local
 systems.
 
-Project configuration include a "Workstation Setup" section where
-administrators declare a number of simple commands (and directory
-contexts) that will run a project's setup. These commands are *not*
-shell interpolated, and are *not* meant to provision the development
-environment (e.g. install system packages or modify configuration
-files in `~/`). Use these commands to clone a repository, generate
-index files, and/or run a test build.
+Project configuration include a ["Workstation Setup" section](../Project-Configuration/Project-and-Distro-Settings.md#virtual-workstation-commands)
+where administrators declare a number of simple commands (and directory
+contexts) that will run a project's setup. These commands are *not* shell
+interpolated, and are *not* meant to provision the development environment (e.g.
+install system packages or modify configuration files in `~/`). Use these
+commands to clone a repository, generate index files, and/or run a test build.
```

Comment:
If we have a handy tight example from server or something we can add that might be nice.

Reply:
I didn't write this text, so I'm inclined to leave it. The server workstation commands aren't super generalizable since it's a clone of a specific repo, then just runs a script in that repo.